data/transport missions.txt

```diff
@@ -1868,6 +1872,310 @@ mission "Terraforming 5"
+				accept
+
+
+
+mission "Terraforming 11"
+	landing
+	name "Terraforming Tundra"
+	description "Return to <destination> with the Republic representative."
+	destination "Tundra"
+	passengers 1
+	to offer
+		has "Terraforming 10: done"
+	
+	on offer
+		conversation
+			`"Wait here," the Republic representative says curtly. Several hours later, he returns, not looking very happy. "What a waste of time. Parliament has declined my request. I don't much agree with this, but you will need to crash an asteroid into Tundra if you wish to achieve your goal." Before you ready for launch, you send a message to Amy and Nolan telling them that an asteroid will be used after all. You don't receive an immediate response.`
+				accept
+	
+	on enter "Cebalrai"
+		dialog
+			`Your ship is targeted by pirate fleets when you enter the system. Before you engage, you receive a message from Amy.`
+			`"Captain, these pirate ships have been occupying the system for days. You must take them out if we are to safely crash an asteroid into Tundra."`
+	npc evade
+		government "Bounty Hunter"
+		personality nemesis staying target
+		system "Cebalrai"
+		fleet "Marauder fleet I"
+		fleet "Large Southern Pirates"
+
+
+
+mission "Terraforming 12"
+	landing
+	name "Terraforming Tundra"
+	description "Plant a thruster on an asteroid to guide it into a collision with the dormant supervolcano on Tundra."
+	cargo "thruster equipment" 10
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 11: done"
+	
+	on offer
+		payment 300000
+		conversation
+			`The pirates that occupied the system did not make an effort to land on Tundra, but they did harass any merchant ships that passed through the system. The government of Tundra pays you <payment> for ridding the system of the pirates, a pleasantly unexpected reward.`
+			`	You gather with the scientists in the spaceport bar. The Tundra native informs you that all residents close enough to the volcano to be in danger have been evacuated. "All we need to do now is find the right asteroid" After only a short time of searching, Amy finds the perfect candidate.`
+			`	"Just like last time, Captain," she says. Spaceport workers load a small thruster and the equipment you will need to mount it on the asteroid into your ship.`
+				accept
+	
+	npc assist
+		government "Uninhabited"
+		personality derelict fleeing uninterested waiting pacifist mute
+		ship "Asteroid" "Target Asteroid"
+	
+	on complete
+		payment 200000
+		event "terraforming timer 2" 365 730
+		event "terraforming Tundra"
+		conversation
+			`A plume of smoke forms in a mushroom cloud shape above where the asteroid hit. The scientists are viewing their instruments in anticipation for what happens next.`
```

Comment:
I think we should rewrite this first sentence. It's not like the smoke forms as soon as the player lands, it's a bit before that, when he arms the asteroid.